lapce-app/src/command.rs

```diff
@@ -191,6 +191,10 @@ pub enum LapceWorkbenchCommand {
     #[strum(message = "Reveal in Panel")]
     RevealInPanel,
 
+    #[strum(serialize = "open_in_git_bub")]
+    #[strum(message = "Open In GitHub")]
+    OpenInGitHub,
```

Comment:
Can you please rename it to  `SourceControlOpenActiveFileRemoteUrl`

similar to this:
https://github.com/lapce/lapce/blob/master/lapce-app/src/command.rs#L490